string prompt
# https://github.com/openai/whisper/blob/main/whisper/tokenizer.py for list of languages
string lang
---
string text